### PR TITLE
build: Add CI

### PR DESCRIPTION
**Pull request by ryanwi**

### CODE REVIEW

**Review by jpsantosbh — APPROVED** [review]

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
```
Comment (ryanwi) [inline .github/workflows/node.js.yml:20]:
```suggestion
```

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
```
Comment (ryanwi) [inline .github/workflows/node.js.yml:19]:
```suggestion
    - name: Use Node.js
```

File: .github/workflows/node.js.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
```
Comment (ryanwi) [inline .github/workflows/node.js.yml:22]:
```suggestion
        node-version: 20.x
```

File: .github/workflows/node.js.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
```
Comment (ryanwi) [inline .github/workflows/node.js.yml:3]:
```suggestion
```

**Comment by ryanwi** [comment]
There's no tests tho, so not sure if we care yet?

**Pull request merged**